app/views/notices/shop_employee_notices/initial_employee_plan_selection_confirmation.html.erb

```diff
@@ -0,0 +1,46 @@
+<%= render partial: 'notices/shared/cover', :locals => {notice: notice}%>
+<%= render partial: 'notices/shared/logo' %>
+<%= render partial: 'notices/shared/address', :locals => {notice: notice} %>
+<%= render partial: 'notices/shared/date', :locals => {notice: notice} %>
```

Comment:
Please Add 5 break statements before rendering the date partial. The date should be below 3.67" mark.